src/app/services/account.js

```diff
@@ -3,33 +3,63 @@ import lisk from 'lisk-js';
 app.factory('Account', function ($rootScope) {
   this.account = {};
 
+  const equals = (ref1, ref2) => {
+    /* eslint-disable eqeqeq */
+
+    if (ref1 == undefined && ref2 == undefined) {
+      return true;
+    }
+    if (typeof ref1 !== typeof ref2 || (typeof ref1 !== 'object' && ref1 != ref2)) {
+      return false;
+    }
+
+    const props1 = (ref1 instanceof Array) ? ref1.map((val, idx) => idx) : Object.keys(ref1).sort();
```

Comment:
What this function does is to make sure we're updating list only when a real change is happening. so it doesn't perform a crucial task. though to meet 90+% it makes sense to cover it.